src/providers/dataProvider/index.ts

```diff
@@ -0,0 +1,45 @@
+import localStorageProvider from 'ra-data-local-storage';
+import { withLifecycleCallbacks } from 'react-admin';
+import constants from '../../constants';
+import { AuditType, trackEvent } from '../../utils';
+import users from './users';
+
+export const dataProvider = localStorageProvider({
+	localStorageKey: constants.LOCAL_STORAGE_DB_KEY,
+	defaultData: {
+		users,
+	},
+});
+
+export const dataProviderWithLifeCycle = withLifecycleCallbacks(dataProvider, [
+	{
```

Comment:
@salony75way - we don't need to audit `get` events, only data changes.

Reply:
Done.